spot_driver/launch/point_cloud_xyz.launch.py

```diff
@@ -37,22 +37,21 @@
 
 import launch_ros.actions
 import launch_ros.descriptions
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration, PathJoinSubstitution
```

Comment:
Have we tested this file?  (I haven't used it in its previous or current state.)

Reply:
Yes it still works but it is very slow because the depth publishing is slow.